shell/platform/android/io/flutter/plugin/mouse/MouseCursors.java

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugin.mouse;
```

Comment:
Is this expected to be used in plugins? I think this would maybe make more sense in `io.flutter.embedding.engine.systemchannels`.